recipes.js

```diff
@@ -1,11 +1,50 @@
 const mongoose = require('mongoose');
+
+const Recipe = require("./recipe-model.js");
```

Comment:
Good! It is good practice to have your model and your queries in separate files, they are linked correctly!